.github/workflows/test.yaml

```
@@ -16,5 +16,31 @@ jobs:
+      matrix:
+        test_type: [unit, integration, acceptance]
+        partition: [1, 2, 3]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          name: dist
+          path: dist
+      - uses: ./.github/actions/setup-app
+      - run: |
+        npm run test:ember -- \
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 43: could not find expected ':'

(syntax-check)